triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/DevOnly.scala

```diff
@@ -105,11 +103,8 @@ class DevOnly
           )
 
           // Determine ACS for this test run's setup
-          queryResult <- runner.queryACS()
-          (acs, _) = queryResult
+          (acs, _) <- runner.queryACS()
 
-          // 1 for ledger create command completion
-          // 1 for ledger create command completion
```

Comment:
It appears that these 2 additional creates were a side effect of using sandbox-on-x?